[ ERROR ] Error in file 'C:\Users\csaba\Documents\Projects\qa-suite-rec\robot-framework-python-tests\tests\api_tests\data_driven_launch_tests.robot' on line 2: Resource file '..\..\keywords.robot' does not exist.
